tests/spread/core22/linters/library-missing/expected_linter_output.txt

```diff
@@ -2,6 +2,6 @@ Running linters...
 Running linter: classic
 Running linter: library
 Lint warnings:
-- library: linter-test: missing dependency 'libcaca.so.0'. (https://snapcraft.io/docs/linters-library)
-- library: linter-test: missing dependency 'libslang.so.2'. (https://snapcraft.io/docs/linters-library)
+- library: linter-test: missing dependency 'libcaca.so.0'. (provided by 'libcaca0') (https://snapcraft.io/docs/linters-library)
```

Comment:
i am not sure it will be easy to correlate provided by with a possible stage-packages entry. Should be fine if the documentation mentions that.